.circleci/config.yml

```diff
@@ -463,14 +463,17 @@ jobs:
           name: '⭐ amp release ⭐'
           command: node --unhandled-rejections=strict build-system/release-workflows/build-release.js
       - teardown_vm
-  archive_release:
+  upload_release:
     executor:
       name: node-docker-medium
     steps:
       - setup_vm
       - run:
-          name: '⭐ Archive Release Artifacts ⭐'
-          command: node --unhandled-rejections=strict build-system/release-workflows/archive-release.js
+          name: 'Store Service Key in JSON File'
+          command: echo "$GCLOUD_SERVICE_KEY" > /tmp/amp-cdn-serving.json
```

Comment:
This might be a nono

Reply:
This was a test to make sure printing the secret didn't work :)

https://app.circleci.com/pipelines/github/ampproject/amphtml/17623/workflows/cd672357-53ac-4bd2-8321-17ae00aaf5b5/jobs/323178/parallel-runs/0/steps/0-101

![image](https://user-images.githubusercontent.com/26553114/136989878-c07b29b6-3e5b-4419-8430-93c557baf10a.png)

Reply:
I think if you `gcloud auth` first, you don't have to pass a keyfile when doing `new Storage()`. Could you try? No key in the code would be ideal 👍